dashboard/test/models/blockly_test.rb

```diff
@@ -567,6 +567,49 @@ def create_custom_block(name, pool, block_text, args, category: 'Events')
     assert_equal expected_localized_block_xml, localized_block_xml
   end
 
+  test 'localized_markdown_with_placeholder_texts' do
```

Comment:
I don't think it would be too much hassle to add `long_instructions` and `short_instructions` to the level we are creating in this test, and then call them to verify that those properties are actually get translated.

Reply:
Added 👍